index.d.ts

```diff
@@ -390,6 +390,7 @@ export namespace Places {
 		 */
 		ownerId?: string;
 		mediaCount: number;
+		satellite: PlaceSatelliteImage | null;
```

Comment:
Preco `PlaceSatelliteImage` ked v `PlaceDetail.ts` je to `SatelliteImage` ?

Reply:
Force pushed. Som pozeral na definiciu v danom subore, ale v `index.d.ts` su este `PlaceGeometry` a `PlaceOpeningHours` co ma zmiatlo. Fixnute.